src/asciidoc/docs/publish/customDomains.adoc

```diff
@@ -0,0 +1,84 @@
+= Using custom domains
+:page-slug: custom-domains
+:page-description: How to configure your Koji to use a custom domain.
+
+When you're ready to go live with your template, you might want to use a custom domain name in place of the name provided by Koji.
+A custom domain can have the advantage of being easier to remember, and it usually just looks better.
+
+Koji has a few subdomains that you can use at no cost and with no additional configuration.
+These include:
+
+* withkoji.com
+* koji.sh
+* koji.game
+
+You can also use a domain name that you purchase from somewhere like *Google Domains* or *Namecheap*.
+
+== Using a Koji subdomain
+
+. Open the Koji and click the Koji button.
+
+. Go to *Settings > Custom domains*.
+
+. If you have already configured a domain and want to change it, follow the steps for <<_releasing_a_domain>> and then come back to this step.
```

Comment:
```suggestion
. If you have already configured a domain and want to change it, follow the steps for <<_releasing_a_domain, releasing a domain>>.
. Click *New*.
```
This is a contextual reference, so we should format it as such. Also, it's a bit confusing to say "come back to this step." It seems like the user is returned to the same place in the UI after releasing a domain, so we can just leave it at that. The next step is the same whether you have just released a domain, or not. Change in the next procedure, as well.